Reportedly/Modules/LoginEmail/LoginEmailInteractor.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 protocol LoginEmailInteractorInput: AnyObject {
-
+    
```

Comment:
Same here. Couldn't we have line breaks instead of spaces?

Reply:
same comment as https://github.com/nimblehq/reportedly-ios/pull/3#discussion_r531407210